examples/saving_and_loading_data/load_adapt_fits_into_map.py

```diff
@@ -0,0 +1,70 @@
+"""
+========================
+Reading ADAPT FITS Files
+========================
+
+This example demonstrates how to load data from the
+Air Force Data Assimilative Photospheric Flux Transport (ADAPT) model into a list of `sunpy.map.Map`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.io import fits
+
+import sunpy.map
+from sunpy.coordinates.sun import carrington_rotation_time
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###############################################################################
+# First we will download an ADAPT FITS file.
+# To do this we will use the `sunpy.net.Fido` search interface to search for
+# ADAPT data for Carrington Rotation 2193, and the first longitude type (0)
+# which means the data will be in a Carrington centered coordinate frame.
```

Comment:
What does centered mean here? Should this just be Carrington frame?

Reply:
Good point. The website said centered so I used it but yeah, it's just a CR frame if I understand this image correctly:

![adapt-2](https://github.com/user-attachments/assets/f00aec7d-ccc3-4627-8601-683d87503981)